lib/discourse_logstash_logger.rb

```diff
@@ -83,9 +83,14 @@ def add_with_opts(severity, message = nil, progname = nil, opts = {}, &block)
       #
       # In theory we could get logster to include the exception class and message in opts but logster currently does not
       # need those options so we are parsing it from the message for now and not making a change in logster.
-      if progname == "web-exception" && message =~ /\A([^\(\)]+)\s{1}\(([\s\S]+)\)/
-        event["exception.class"] = $1
-        event["exception.message"] = $2.strip
+      if progname == "web-exception"
+        # `Logster.store.ignore` is set in the logster initializer and is an array of regex patterns.
```

Comment:
https://github.com/discourse/discourse/blob/e364ed2ad1b4d12aac72afd82bc6eb3e30bd8605/config/initializers/100-logster.rb#L25-L58

This is the list of configured regexps